godtools/Views/TractElements/BaseTractElement.swift

```diff
@@ -197,12 +197,12 @@ class BaseTractElement: UIView {
     
     func getPreviousElement() -> BaseTractElement? {
         guard let index = getElementPosition() else {
-            return nil
+            return self.elements?.last
```

Comment:
Previously, if it was impossible to get the element position (because the list was empty or the element wasn't added to the list yet) it returned nil, but, because it is currently processing the element (so the position is not set yet but will be set when the element is processed it gets the last element of the list (which will be the previous one.